modules/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/GeoIpDownloaderTaskExecutor.java

```diff
@@ -398,7 +398,7 @@ private void stopTask(Runnable onFailure) {
             ActionListener.runAfter(listener, () -> {
                 IndexAbstraction databasesAbstraction = clusterService.state()
                     .metadata()
-                    .getProject()
+                    .getDefaultProject()
```

Comment:
The difference is that `getDefaultProject()` does not throw when there are multiple projects in the cluster state. It helps unblock the work for the multi-project smoke IT. The downside is that it makes the failure less visible. Given the complexity involved in fixing GeoIp executor, I feel this is the right trade-off similar to how we use it for ML reset and system index migration.